config/config.go

```diff
@@ -47,6 +47,7 @@ type Config struct {
 	TurnAddress    string `default:":3478" required:"true" split_words:"true"`
 	TurnStrictAuth bool   `default:"true" split_words:"true"`
 	TurnPortRange  string `split_words:"true"`
+	TurnExternal   bool   `default:"false"`
```

Comment:
Also, instead of a boolean flag, it probably would be cool, if you could specify the address of the external turn server, so that it can be hosted on a separate server.

Reply:
See 799f05e4e6a8b45e6b1aab1faacbd4d743206fc5

Reply:
@jmattheis it's essentially `ExternalIP` config that points to the location of external TURN server, right? it's the only place where it is used, in `ws/room.go` in `addresses()`

In my setup coturn and screego are not in the same network (but both have the same public IP), and this works 🙂